.travis.yml

```diff
@@ -33,14 +33,19 @@ matrix:
   - rvm: 2.4.1
     bundler_args: --without system_tests development
     env: PUPPET_VERSION="~> 4.0" CHECK=build DEPLOY_TO_FORGE=yes
-  - rvm: 2.4.0
+  - rvm: 2.4.2
     bundler_args: --without development
-    env: PUPPET_VERSION="~> 4.0" CHECK=acceptance BEAKER_set="docker/debian-8" bundle exec rake acceptance
+    env: PUPPET_VERSION="~> 5.0" CHECK=acceptance BEAKER_set="docker/debian-8" bundle exec rake acceptance
     services: docker
     sudo: required
-  - rvm: 2.4.0
+  - rvm: 2.4.2
     bundler_args: --without development
-    env: PUPPET_VERSION="~> 4.0" CHECK=acceptance BEAKER_set="docker/centos-7" bundle exec rake acceptance
+    env: PUPPET_VERSION="~> 5.0" CHECK=acceptance BEAKER_set="docker/centos-7" bundle exec rake acceptance
+    services: docker
+    sudo: required
+  - rvm: 2.4.2
+    bundler_args: --without development
+    env: PUPPET_VERSION="~> 5.0" CHECK=acceptance BEAKER_set="docker/debian-9" bundle exec rake acceptance
```

Comment:
```beaker-puppet_install_helper``` uses a different syntax. How this even worked in the past is a mystery to me but if you read https://github.com/puppetlabs/beaker-puppet_install_helper#run_puppet_install_helper then it suggests you need explicit version numbers.

Reply:
to clarify: This never worked and was a copy and paste issue